tests/test_resume_arm_time.py

```
@@ -265,35 +265,42 @@ def fail_read(cmd, **kwargs):
+    marker_line = next(l for l in block.splitlines() if l.startswith("# taOSmd-resume: "))
+    marker = marker_line[len("# taOSmd-resume: "):]
+
     initial_crontab = (
         f"# taOSmd-resume: {marker}\n"
-        + f"7 0 18 8 * /usr/bin/python3 /home/jay/.taos-fleet-tools/resume_arm_time.py --fire primary {armed_at}\n"
+        + f"7 0 18 8 * /usr/bin/python3 {resume_arm_time._HELPER_PATH} --fire primary {fire_dt.isoformat()}\n"
         + WATCHER_LINE
     )
     state = _install_fake_crontab(tmp_path, monkeypatch, initial_crontab)
 
     proc = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)